src/mon/MonCommands.h

```diff
@@ -628,7 +628,7 @@ COMMAND("osd erasure-code-profile ls", \
 	"list all erasure code profiles", \
 	"osd", "r", "cli,rest")
 COMMAND("osd set " \
-	"name=key,type=CephChoices,strings=full|pause|noup|nodown|noout|noin|nobackfill|norebalance|norecover|noscrub|nodeep-scrub|notieragent|sortbitwise|require_jewel_osds", \
+	"name=key,type=CephChoices,strings=full|pause|noup|nodown|noout|noin|nobackfill|norebalance|norecover|noscrub|nodeep-scrub|notieragent|sortbitwise|require_jewel_osds|require_kraken_osds", \
```

Comment:
probably you can reference this line: https://github.com/ceph/ceph/blob/9bef6cf/src/mon/OSDMonitor.cc#L6355, in the commit message.

i am not able to identify the piece of code referenced by https://github.com/ceph/ceph/pull/9248#issuecomment-221421724 anymore.